pkg/sidecars/lb_sidecar/handlers.go

```diff
@@ -0,0 +1,160 @@
+package sidecarserv
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/inspr/inspr/cmd/insprd/memory/tree"
+	"github.com/inspr/inspr/pkg/environment"
+	"github.com/inspr/inspr/pkg/ierrors"
+	metautils "github.com/inspr/inspr/pkg/meta/utils"
+	"github.com/inspr/inspr/pkg/rest"
+	"go.uber.org/zap"
+)
+
+var logger *zap.Logger
+
+func init() {
+	logger = zap.NewNop()
+
+	// logger, _ = zap.NewProduction(zap.Fields(zap.String("section", "loadbalencer-sidecar")))
+}
+
+// writeMessageHandler handles requests sent to the write message server
+func (s *Server) writeMessageHandler() rest.Handler {
+	return func(w http.ResponseWriter, r *http.Request) {
+		logger.Info("handling message write")
+
+		channel := strings.TrimPrefix(r.URL.Path, "/")
+
+		if !environment.OutputChannnelList().Contains(channel) {
+			logger.Error("channel " + channel + " not found in output channel list")
+			insprError := ierrors.NewError().
+				BadRequest().
+				Message("channel '%s' not found", channel)
+
+			rest.ERROR(w, insprError.Build())
+			return
+		}
```

Comment:
I was wondering if maybe we could change the list to be a map, actually, so that the testing is O(1). I know it seems stupid now, but with thousands of messages it may actually have a lot of impact.

Reply:
It isn't stupid at all. I'll create a tech dep for it